### DISCUSSION TITLE
Action on click inside tooltip

### DISCUSSION THREAD

**Discussion by rstanislav** [atomiks/tippyjs#1067] [Q&A]
Hello! Tried to search in issues and discussions, but didnt find any info about this..
Is it possible to exec action on click **inside** tooltip ? (i need to copy data to clipboard, so i need to exec javascript code)
The is prop - onClickOutside, in my case i need something like onClickinside
Tippy is defined like this in my project:

```
tippy('[data-tippy-content]', {
                animation: 'fade',
                interactive: true,
                theme: 'material',
                followCursor: 'initial',
                trigger: "click",
                appendTo: () => document.body,
                plugins: [window.followCursor],
            });
```

**Comment by microwapp** [comment]
You could use the below method to add a listener to the element to trigger the function
`onCreate(instance) {
		instance.popper
			.querySelector('.tippy__eleName)
			.addEventListener('click', () => console.log('clicked'));
}`

Also, please enable `allowHTML: true` otherwise it will throw errors.
